### PR TITLE
Add  Hello World in Groovy language

### PR DESCRIPTION
**Pull request by Veekshitha11**
hello world in groovy

### CODE REVIEW

**Comment by abhisek247767** [comment]
please check guidelines(mention programming language)..in this repos all PR please do changes.

**Comment by Veekshitha11** [comment]
Sir I have already mentioned the programming language

**Comment by abhisek247767** [comment]
no. you don't  mention on that file with comment.

File: helloworld.groovy
Comment (Veekshitha11) [inline helloworld.groovy:1]: Hello world in groovy programming language

**Comment by Veekshitha11** [comment]
Sir I have mentioned that now'

**Comment by Veekshitha11** [comment]
sir I have made changes according to the guidelines can you please the merge the pull request now
Thank you sir

**Pull request merged**